multi-node/aws/pkg/cluster/cluster.go

```diff
@@ -156,25 +157,18 @@ func (c *Cluster) Create(tlsConfig *TLSConfig) error {
 		})
 	}
 
-	if c.cfg.AvailabilityZone != "" {
-		parameters = append(parameters, &cloudformation.Parameter{
-			ParameterKey:     aws.String(parAvailabilityZone),
-			ParameterValue:   aws.String(c.cfg.AvailabilityZone),
-			UsePreviousValue: aws.Bool(true),
-		})
-	}
-
 	tmplURL := fmt.Sprintf("%s/template.json", c.cfg.ArtifactURL)
-	return createStackAndWait(cloudformation.New(c.aws), c.stackName(), tmplURL, parameters)
+	return createStackAndWait(cloudformation.New(session.New(c.aws)), c.stackName(), tmplURL, parameters)
```

Comment:
I had to upgrade deps to handle ELB and it's now necessary to create a session instance.